geowebcache/arcgiscache/src/test/java/org/geowebcache/arcgis/layer/ArcGISCacheLayerTest.java

```diff
@@ -0,0 +1,53 @@
+package org.geowebcache.arcgis.layer;
```

Comment:
Appreciate the test, can you add project header this this file please.

Reply:
You got it. Just added.